src/HSE/Util.hs

```diff
@@ -16,7 +16,6 @@ import HSE.Type
 import Data.Functor
 import Prelude
 
-
```

Comment:
Deleting my beautiful 2 lines after imports...

Reply:
Sorry, I will restore. When I finally realized you have a pattern I've tried to respect that.

Reply:
✔️